optional/gabz_config.lua

```diff
@@ -0,0 +1,1783 @@
+	},
+	vagner = {
+		shop = 'luxury',
+	},
+	visione = {
+		shop = 'luxury',
+	},
+	voltic = {
+		shop = 'luxury',
+	},
+	voltic2 = {
+		shop = 'luxury',
+	},
+	xa21 = {
+		shop = 'luxury',
+	},
+	zentorno = {
+		shop = 'luxury',
+	},
+	zorrusso = {							--DLC
+		shop = 'luxury',
+	},
+	-- Vans
+	bison = {
+		shop = 'pdm',
+	},
+	bobcatxl = {
+		shop = 'pdm',
+	},
+	burrito3 = {
+		shop = 'pdm',
+	},
+	gburrito2 = {
+		shop = 'pdm',
+	},
+	rumpo = {
+		shop = 'pdm',
+	},
+	journey = {
+		shop = 'pdm',
+	},
+	minivan = {
+		shop = 'pdm',
+	},
+	minivan2 = {
+		shop = 'pdm',
+	},
+	paradise = {
+		shop = 'pdm',
+	},
+	rumpo3 = {
+		shop = 'pdm',
+	},
+	speedo = {
+		shop = 'pdm',
+	},
+	speedo4 = {
+		shop = 'pdm',
+	},
+	surfer = {
+		shop = 'pdm',
+	},
+	youga3 = {
+		shop = 'pdm',
+	},
+	youga = {
+		shop = 'pdm',
+	},
+	youga2 = {
+		shop = 'pdm',
+	},
+	youga4 = { 	 	--DLC +set sv_enforceGameBuild 2545
+		shop = 'pdm',
+	},
+	mule5 = { 	 	--DLC +set sv_enforceGameBuild 2545
+		shop = 'pdm',
+	},
+	-- Utility
+	sadler = {
+		shop = 'pdm',
+	},
+	guardian = {
+		shop = 'pdm',
+	},
+	slamtruck = {
+		shop = 'pdm',
+	},
+	warrener2 = {							--DLC +set sv_enforceGameBuild 2372
+		shop = 'pdm',
+	},
+		-- Boats
+	squalo = {
+		shop = 'boats',
+	},
+	marquis = {
+		shop = 'boats',
+	},
+	seashark = {
+		shop = 'boats',
+	},
+	seashark2 = {
+		shop = 'boats',
+	},
+	seashark3 = {
+		shop = 'boats',
+	},
+	jetmax = {
+		shop = 'boats',
+	},
+	tropic = {
+		shop = 'boats',
+	},
+	tropic2 = {
+		shop = 'boats',
+	},
+	dinghy = {
+		shop = 'boats',
+	},
+	dinghy2 = {
+		shop = 'boats',
+	},
+	dinghy3 = {
+		shop = 'boats',
+	},
+	dinghy4 = {
+		shop = 'boats',
+	},
+	suntrap = {
+		shop = 'boats',
+	},
+	speeder = {
+		shop = 'boats',
+	},
+	speeder2 = {
+		shop = 'boats',
+	},
+	longfin = {
+		shop = 'boats',
+	},
+	toro = {
+		shop = 'boats',
+	},
+	toro2 = {
+		shop = 'boats',
+	},
+	-- helicopters
+	buzzard2 = {
+		shop = 'air',
+	},
+	frogger = {
+		shop = 'air',
+	},
+	frogger2 = {
+		shop = 'air',
+	},
+	maverick = {
+		shop = 'air',
+	},
+	swift = {
+		shop = 'air',
+	},
+	swift2 = {
+		shop = 'air',
+	},
+	seasparrow = {
+		shop = 'air',
+	},
+	seasparrow2 = {
+		shop = 'air',
+	},
+	seasparrow3 = {
+		shop = 'air',
+	},
+	supervolito = {
+		shop = 'air',
+	},
+	supervolito2 = {
+		shop = 'air',
+	},
+	volatus = {
+		shop = 'air',
+	},
+	havok = {
+		shop = 'air',
+	},
+	-- Planes
+	duster = {
+		shop = 'air',
+	},
+	luxor = {
+		shop = 'air',
+	},
+	luxor2 = {
+		shop = 'air',
+	},
+	stunt = {
+		shop = 'air',
+	},
+	mammatus = {
+		shop = 'air',
+	},
+	velum = {
+		shop = 'air',
+	},
+	velum2 = {
+		shop = 'air',
+	},
+	shamal = {
+		shop = 'air',
+	},
+	vestra = {
+		shop = 'air',
+	},
+	dodo = {
+		shop = 'air',
+	},
+	howard = {
+		shop = 'air',
+	},
+	alphaz1 = {
+		shop = 'air',
+	},
+	nimbus = {
+		shop = 'air',
+	},
+	brioso3 = {          --DLC +set sv_enforceGameBuild 2699 (and below)
+        shop = 'pdm',
+    },
+    conada = {
```

Comment:
This is because my formatter doesn't like comments next to a `{` for some reason. 
example; https://i.imgur.com/YuTfIs2.png - will just move all comments to the `shop = 'pdm',` line